github-actions/start-release/templates/connector_detail.md

```diff
@@ -19,12 +19,12 @@ Minimum Product Version: {{connector.min_phantom_version}}
 {%- if connector.configuration %}
 
 ### Configuration Variables
-The below configuration variables are required for this Connector to operate.  These variables are specified when configuring a {{connector.product_name}} asset in SOAR.
+This table lists the configuration variables required to operare {{connector.name}}. These variables are specified when configuring a {{connector.product_name}} asset in SOAR.
 
 VARIABLE | REQUIRED | TYPE | DESCRIPTION
 -------- | -------- | ---- | -----------
 {%- for name, parameters in connector.configuration.items() %}
-{%- if name != "ph" %}
+{%- if name != "ph" and ("visibility" not in parameters or parameters.visibility|length > 0) %}
```

Comment:
Would this have prevented this issue? https://github.com/splunk-soar-connectors/msgraphforoffice365/pull/44/files#r1854866249

Reply:
it wouldn't but the condition should change to include parameters.data_type != "ph" as long as a ph data_type should be hidden.

Reply:
Then let's change the condition? The `ph` data type exists solely to make UI forms look better. It's a nasty hack